src/u_scanner.c

```diff
@@ -250,7 +292,7 @@ boolean U_GetNextToken(u_scanner_t* scanner, boolean   expandState)
   cur   = scanner->data[scanner->scanPos++];
 
   // Determine by first character
-  if(cur == '_' || cur == '-' || (cur >= 'A' && cur <= 'Z') || (cur >= 'a' && cur <= 'z'))
+  if(cur == '_' || (cur >= 'A' && cur <= 'Z') || (cur >= 'a' && cur <= 'z'))
```

Comment:
This is already spec incompliant, as `TK_Identiier` tokens are expected to start with a letter.

Reply:
It's the same in PrBoom+: https://github.com/coelckers/prboom-plus/blob/969515162c5aebea4ad7a125ee178dcad3d576ad/prboom2/src/scanner.cpp#L252 I think we should keep it.

Reply:
Okay, sure.